tests/strict-equality.test.js

```diff
@@ -1,70 +1,58 @@
 const describe = require('mocha').describe
 const it = require('mocha').it
 const expect = require('chai').expect
-const strictequal = require('../strict-equality')
+const strictEqual = require('../strict-equality')
 
 
 describe('strict-equality', () => {
   describe('compare', () => {
```

Comment:
The idea with `describe` is to provide some structure around where the code that is being tested lives. So you'd follow a pattern sure as a describe for the module, then within that a describe for each function in the module. This way when looking at the test output you immediately know where to find the function that is causing a failing test. This becomes incredibly necessary in real projects which have dozens of folders and 100s of functions.